tasks-june/script.js

```diff
@@ -0,0 +1,112 @@
+/*	TASK 1
+	Avem 2 inputuri de tip text intr-un formular care reprezinta numele si prenumele. 
+	Sa se afiseze cu ajutorul unui alert un mesaj de salut.
+*/
+function greet() {
+	alert(`Hello ${nume.value} ${prenume.value}. How are you today`);
+}
+
+
+/*	TASK 2
+	Avem un dropdown cu 3 optiuni. 
+	Cu ajutorul unui buton sa se stearga optiunea selectata din dropdown.
+*/
+function deleteOp() {
+	let toDelete = dropdown.value;
+	let allOptions = document.querySelectorAll('option');
+	for (let item of allOptions) {
+		if (toDelete === item.value) {
+			item.remove();
+		}
+	}
+}
+
+
+/*	TASK 3
+	Avem un element de tip <input> pentru a introduce un numar ce reprezinta 
+	raza unei sfere. Sa se afiseze volumul corespunzator razei.
+*/
+function getVolume() {
+	console.log(radius.value);
+	let volume = 4 / 3 * Math.PI * Math.pow(radius.value, 3);
+	result.textContent += volume;
+}
+
+
+/*	TASK 4
+	Sa se faca highlight (cu culoarea orange) pe cuvintele cu stilul bold cand se face hover pe text. 
+	Altfel, revine la stilul initial (adica bold).
+*/
+let boldWords = [];
+window.onload = function () {
+	boldWords = document.getElementsByTagName('b');
+}
+
+function hightlight() {
+	for (let word of boldWords) {
+		word.style.backgroundColor = 'orange';
+		word.style.fontWeight = 100;
+	}
+}
+
+function reset() {
+	for (let word of boldWords) {
+		word.style.backgroundColor = 'white';
+		word.style.fontWeight = 700;
+	}
+}
```

Comment:
you can improve a little bit. Did you notice the duplicate code?
```suggestion
function setStyleOnBoldWords(backgroundColor, fontWeight) {
        for (let word of boldWords) {
	        word.style.backgroundColor = backgroundColor;
		word.style.fontWeight = fontWeight;
	}
}

function hightlight() {
	setStyleOnBoldWords('orange', 100);
}

function reset() {
	setStyleOnBoldWords('white', 700);
}
```

Reply:
in this case, you only modify the parameters